src/main/java/core/basesyntax/IFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface IFigure {
```

Comment:
Interface names should not start with 'I'. It's a Java convention typically used in C#, but not in Java. Consider renaming the interface to something more descriptive, such as 'Figure' if it represents a general figure or 'Drawable' if it's meant to represent drawable objects.